docs/manage/configuration.md

```diff
@@ -0,0 +1,298 @@
+When changes are made to the fragment, those changes are automatically carried to all robots that include the fragment in their config.
+
+You can add a fragment to a robot's config and also add other configuration outside the fragment.
+For example, if you have multiple identical rovers but one has an arm attached, you can add the rover configuration fragment (including the motors and base components), and then configure the arm on just that one rover.
+
+To add a fragment to a robot:
+
+- Go to the **FRAGMENTS** sub-tab of your robot's **CONFIG** tab on the [Viam app](https://app.viam.com).
+- Look through the list of available fragments and click **ADD** next to any fragments you want to add to your robot.
+- Click **Save Config** at the bottom of the screen.
+
+{{% alert title="Note" color="note" %}}
+
+The components or other resources included in the fragment will *not* appear in the **COMPONENTS** sub-tab or in the `components` section of your config.
+You will simply see the fragment ID in the `fragments` section of your config, but all the resources *will* appear on the **CONTROL** tab when your robot comes on line, and they will be accessible from SDK code as regular resources of your robot.
+
+{{% /alert %}}
+
+To create a fragment, go to [app.viam.com/fragments](https://app.viam.com/fragments).
+
+For an example of adding a fragment to a robot, see the [Viam Rover fragment tutorial](/try-viam/rover-resources/rover-tutorial-fragments/).
+<!--
+## Auth/network
+-->
+
+## Frame system
+
+The frame system holds reference frame information for the relative position of components in space.
+
+Configure a frame for a given component on its panel on the **COMPONENTS** tab, then switch to the **FRAME SYSTEM** tab to visualize the relative positions.
+Find more information in the [frame system documentation](/services/frame-system/).
+
+## Troubleshooting
+
+If you run into issues, here are some things to try:
+
+- Check the **LOGS** tab to check for any errors or other info from `viam-server`.
+- You can also view logs locally by running the following command on the robot:
+  
+  {{< tabs >}}
+  {{% tab name="Linux" %}}
+
+  ```bash
+  sudo journalctl --unit=viam-server
+  ```
+
+  {{% /tab %}}
+  {{% tab name="macOS" %}}
+
+  ```bash
+  cat $(brew --prefix)/var/log/viam.log
+  ```
+
+  {{% /tab %}}
+  {{< /tabs >}}
+
+- Make sure all configured components are actually saved to your config.
+   If they aren't, you will see an **Unsaved Changes** note next to the **Save Config** button at the bottom of the config screen.
+- Try restarting `viam-server` by clicking **Restart** in the upper right corner of the app.
+   It takes a few minutes for the server to shut down and restart.
+- Make sure the issue is not hardware related.
+  Some things to check are that the robot has adequate power, all wires are properly connected, and no chips or other hardware components are shorted or overheated.
+- {{< snippet "social.md" >}}
+
+## Local Setup

```

Comment:
Do you think there's a good place above where we could link to this?

Reply:
We do link to it from the first section